cmd/nerdctl/container_inspect_linux_test.go

```diff
@@ -138,3 +138,17 @@ func TestContainerInspectContainsMounts(t *testing.T) {
 		}
 	}
 }
+
+func TestContainerInspectContainsLabel(t *testing.T) {
```

Comment:
Would we can parallel test?

Reply:
oh yeah, I was thining about it and forgot haha.